src/definitions/raw.json

```diff
@@ -2,28 +2,34 @@
 	"commandName": "raw",
 	"commandDescription": {
 		"en-US": "Tells you what is the datum of this type with this identifier",
-		"fr": "Te dit quelle est la donnée de ce type avec cet identifiant"
+		"fr": "Te dit quelle est la donnée de ce type avec cet identifiant",
+		"pt-BR": "Te diz qual é o Datum desse tipo de identificador"
 	},
 	"typeOptionName": "type",
 	"typeOptionDescription": {
 		"en-US": "Some type",
-		"fr": "Un type"
+		"fr": "Un type",
+		"pt-BR": "Algum tipo"
 	},
 	"identifierOptionName": "identifier",
 	"identifierOptionDescription": {
 		"en-US": "Some identifier",
-		"fr": "Un identifiant"
+		"fr": "Un identifiant",
+		"pt-BR": "Algum identificador"
 	},
 	"help": {
 		"en-US": "Type `/$<commandName> $<typeOptionDescription> $<identifierOptionDescription>` to know what is the datum of `$<typeOptionDescription>` with `$<identifierOptionDescription>`",
-		"fr": "Tape `/$<commandName> $<typeOptionDescription> $<identifierOptionDescription>` pour savoir quel est la donnée d'`$<typeOptionDescription>` avec `$<identifierOptionDescription>`"
+		"fr": "Tape `/$<commandName> $<typeOptionDescription> $<identifierOptionDescription>` pour savoir quel est la donnée d'`$<typeOptionDescription>` avec `$<identifierOptionDescription>`",
+		"pt-BR": "Digite `/$<commandName> $<typeOptionDescription> $<identifierOptionDescription>` pra saber qual é o Datum de `$<typeOptionDescription>` com `$<identifierOptionDescription>`"
```

Comment:
Do you have a translation for "datum"?

Reply:
I didn't know what was that actually; searching here, it's referred as the same word in Portuguese 🤷‍♂️